### PR TITLE
PACA-450: Update migration scripts for synthetic RDA attribution data

### PR DESCRIPTION
**Pull request by KyRobbins**
### Fixes [PACA-450](https://jira.cms.gov/browse/PACA-450)

Update the Synthetic RDA Attribution migration to use Synthea Release 1 attribution data.

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [ ] no PHI/PII is affected by this change

### CODE REVIEW

File: db/migrations/manual/20220211-partially-adj-sythea-rel1-new.sql [hunk 1]
```diff
@@ -9,9 +9,9 @@ DECLARE Large cclf_files.id%TYPE;
 
 -- Adding three rows into cclf_files for three new CCLF files associated to synthea (improved data)
 BEGIN
-INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST990.ZC8Y22.D220201.T1834180', 'TEST990', '2022-02-01', 21, 'Completed', 0) RETURNING id INTO Dev;
-INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST991.ZC8Y22.D220201.T1834180', 'TEST991', '2022-02-01', 21, 'Completed', 0) RETURNING id INTO Small;
-INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST992.ZC8Y22.D220201.T1834180', 'TEST992', '2022-02-01', 21, 'Completed', 0) RETURNING id INTO Large;
+INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST990.ZC8Y22.D220201.T1834180', 'TEST990', '2021-07-01', 21, 'Completed', 0) RETURNING id INTO Dev;
```
Comment (ericbuckley) [inline db/migrations/manual/20220211-partially-adj-sythea-rel1-new.sql]: we need to make this change `s/ZC8Y22/ZC8Y21`.  The last two digits are for year

File: db/migrations/manual/20220211-partially-adj-sythea-rel1-new.sql [hunk 2]
```diff
@@ -9,9 +9,9 @@ DECLARE Large cclf_files.id%TYPE;
 
 -- Adding three rows into cclf_files for three new CCLF files associated to synthea (improved data)
 BEGIN
-INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST990.ZC8Y22.D220201.T1834180', 'TEST990', '2022-02-01', 21, 'Completed', 0) RETURNING id INTO Dev;
-INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST991.ZC8Y22.D220201.T1834180', 'TEST991', '2022-02-01', 21, 'Completed', 0) RETURNING id INTO Small;
-INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST992.ZC8Y22.D220201.T1834180', 'TEST992', '2022-02-01', 21, 'Completed', 0) RETURNING id INTO Large;
+INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST990.ZC8Y21.D220201.T1834180', 'TEST990', '2021-07-01', 21, 'Completed', 0) RETURNING id INTO Dev;
```
Comment (ericbuckley) [inline db/migrations/manual/20220211-partially-adj-sythea-rel1-new.sql]:
```suggestion
INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST990.ZC8Y21.D210701.T1834180', 'TEST990', '2021-07-01', 21, 'Completed', 0) RETURNING id INTO Dev;
```

File: db/migrations/manual/20220211-partially-adj-sythea-rel1-runout.sql [hunk 3]
```diff
@@ -0,0 +1,10173 @@
...
+
+BEGIN;
+DO $$
+DECLARE Dev cclf_files.id%TYPE;
+DECLARE Small cclf_files.id%TYPE;
+DECLARE Large cclf_files.id%TYPE;
+
+-- Adding three rows into cclf_files for three new CCLF files associated to synthea (improved data)
+BEGIN
+INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST990.ZC8Y21.D220201.T1834180', 'TEST990', '2021-09-29', 21, 'Completed', 0) RETURNING id INTO Dev;
```
Comment (ericbuckley) [inline db/migrations/manual/20220211-partially-adj-sythea-rel1-runout.sql]:
```suggestion
INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST990.ZC8Y21.D210929.T1834180', 'TEST990', '2021-09-29', 21, 'Completed', 0) RETURNING id INTO Dev;
```

**Review by ericbuckley — DISMISSED** [review]

**Review by brianburton — DISMISSED** [review]

File: db/migrations/manual/20220211-partially-adj-sythea-rel1-runout.sql [hunk 4]
```diff
@@ -0,0 +1,10173 @@
...
+DO $$
+DECLARE Dev cclf_files.id%TYPE;
+DECLARE Small cclf_files.id%TYPE;
+DECLARE Large cclf_files.id%TYPE;
+
+-- Adding three rows into cclf_files for three new CCLF files associated to synthea (improved data)
+BEGIN
+INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST990.ZC8Y21.D210929.T1834180', 'TEST990', '2021-09-29', 21, 'Completed', 0) RETURNING id INTO Dev;
+INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST991.ZC8Y21.D210929.T1834180', 'TEST991', '2021-09-29', 21, 'Completed', 0) RETURNING id INTO Small;
+INSERT INTO cclf_files (created_at, updated_at, cclf_num, name, aco_cms_id, timestamp, performance_year, import_status, type) VALUES (now(), now(), 8, 'T.BCD.TEST992.ZC8Y21.D210929.T1834180', 'TEST992', '2021-09-29', 21, 'Completed', 0) RETURNING id INTO Large;
```
Comment (katyasoup) [inline db/migrations/manual/20220211-partially-adj-sythea-rel1-runout.sql]: Nit: I'd recommend updating the name fields here to reflect the pattern difference between runout and regular files (see [here](https://github.com/CMSgov/bcda-app/pull/567), tl;dr is replace Y with R e.g. T.BCD.TEST992.ZC8**R**21.D210929.T1834180)

**Review by katyasoup — COMMENTED** [review]
One small comment, otherwise looks good!

File: db/migrations/manual/20220211-partially-adj-sythea-rel1-runout.sql (hunk 4 above)
Comment (ericbuckley) [reply db/migrations/manual/20220211-partially-adj-sythea-rel1-runout.sql]: Oh interesting, thanks @katyasoup I didn't know that about the format

Comment (katyasoup) [reply db/migrations/manual/20220211-partially-adj-sythea-rel1-runout.sql]: @ericbuckley np! it's easy to miss and the `type` field does the heavy lifting, but it is helpful to be able to cross-reference that field as well

**Review by ericbuckley — APPROVED** [review]

**Review by katyasoup — APPROVED** [review]
🎉

**Review by brianburton — APPROVED** [review]

**Review by CosmicSyntax — APPROVED** [review]
🎉

**Pull request merged**